lib/apiclasses.js

```diff
@@ -35,6 +35,7 @@ function StatusMessage(refersTo, description, isError) {
 /** @const */ StatusMessage.UNSPECIFIED = 'UNSPECIFIED';
 /** @const */ StatusMessage.BREAKPOINT_SOURCE_LOCATION =
                 'BREAKPOINT_SOURCE_LOCATION';
+/** @const */ StatusMessage.BREAKPOINT_ACTION = 'BREAKPOINT_ACTION';
```

Comment:
Added a comment for the future and removed the new status message. Once we have the proto inside our project we can avoid these problems.